frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -105,7 +105,7 @@ class Question {
   constructor(
     card: any,
     metadata?: Metadata,
-    parameterValues?: ParameterValues,
+    parameterValues?: ParameterValues | Record<ParameterId, Parameter["value"]>,
```

Comment:
do we have an idea of what the actual types for `Parameter['value']` could be? I see it's `any` in `Parameter` but I'm sure someone has some guidance on how to update that type to be something useful

Reply:
not at the moment, but I decided to reuse `ParameterValuesMap`